core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/metrics/BaseMetricsITest.java

```diff
@@ -103,7 +103,10 @@ public void initMethod() {
         for (TableMetadata tableMetadata : session.getCluster().getMetadata().getKeyspace(session.getLoggedKeyspace())
                 .getTables()) {
             if(tableMetadata.getName().startsWith(DataAccessImpl.TEMP_TABLE_NAME_PROTOTYPE)) {
-                session.execute(String.format("TRUNCATE %s", tableMetadata.getName()));
+                try {
+                    session.execute(String.format("TRUNCATE %s", tableMetadata.getName()));
+                } catch (Exception e) {
```

Comment:
Shouldn't be part of this PR?

Reply:
What did I miss here?

Reply:
Added try {} catch parts.

Reply:
hmm... I have no idea why that change was made in this PR. I can revert the change.